test/display_tests/display.jl

```diff
@@ -0,0 +1,87 @@
+
+using ValidatedNumerics
+using FactCheck
+
+setprecision(Interval, Float64)
+
+facts("displaymode tests") do
+
+    context("Interval") do
+
+        a = 1..2
+        b = -1.1..1.3
+        c = Interval(pi)
+        d = @interval(π)
```

Comment:
I think we need to add some tests with `Interval{BigFloat}` and `DecoratedInterval{BigFloat}`...